.github/workflows/pull_request.yaml

```diff
@@ -26,40 +12,12 @@ jobs:
         with:
           java-version: '12.x'
       - uses: subosito/flutter-action@v1
-      #   with:
-      #     channel: 'stable'
-      # - name: Upgrade flutter
-      #   run: |
-      #     flutter channel stable
-      #     flutter upgrade
-      # - name: Install packages
-      #   run: |
-      #     ls
-      #     flutter pub get
         with:
           flutter-version: '2.0.3'
       - run: flutter pub get
       - name: Lint
         run: |
           flutter analyze
-#      - name: Unit test
-#        run: |
-#          flutter test
-#  build-iOS:
-#    name: Build iOS package
-#    runs-on: macOS-latest
-#    steps:
-#      - uses: actions/checkout@v1
-#      - uses: actions/setup-java@v1
-#        with:
-#          java-version: '12.x'
-#      - uses: subosito/flutter-action@v1
-#        with:
-#          flutter-version: '2.2.0'
-#      - run: flutter pub get
-#      - name: build iOS package
-#        run: |
-#          flutter build ios --release --no-codesign --no-sound-null-safety lib/main.dart
```

Comment:
Are we going to remove the IOS build? it does not work?

Reply:
Again, commented out code. I think Farzad could not get the IOS build to work. 

This PR is only removing code that is currently not running. 
I would like to have the IOS build run as well.

Reply:
Maybe now we can, nothing is lost with an attempt, if it fails, it is commented again 👍

Reply:
iOS build didn't work on the CI, but was working fine locally, so it was removed from CI - FWIW